src/core/snapshot.js

```diff
@@ -9,22 +9,44 @@ const typeOf = { value: Symbol.for('react.test.json') }
 // Clean up result for better looking on CodeMirror
 const cleanUp = (result) => {
   result.actual = result.actual.trim()
-  result.expected = result.expected.trim()
+  result.expected = (result.expected || '').trim()
   result.diff = (result.diff || '').replace(/\n[ ]+\n/g, '\n\n')
```

Comment:
The example works but it seems there's an extra space at the end of the diff, it'd be great if you could just add `.trim()` here.

<img width="206" alt="screen shot 2018-02-04 at 10 43 15" src="https://user-images.githubusercontent.com/1308971/35776651-3e70ec9c-0998-11e8-8b90-17c3fa18fb43.png">

Reply:
👍 Almost did that, but I wasn't sure if it would cause problems.